### PR TITLE
UG Update: Update and add images

### PR DESCRIPTION
**Pull request by JonahhGohh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -227,18 +227,27 @@ The following subsections will elaborate on the specific details of each TutorBu
...
 Displays URL to TutorBuddy's User Guide and a list of all commands and their usages.
 
-![help message](images/helpMessage.png)
+![help message window](images/helpMessageWindow.png)
 
 Format: `help`
 
+To access the help page, do one of the following:
+1. type `help` in the command box 
+2. press <kbd>f1</kbd>
```
Comment (samleewy) [inline docs/UserGuide.md]: This is OCD, but perhaps capital F1?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -227,18 +227,27 @@ The following subsections will elaborate on the specific details of each TutorBu
...
 ##### Listing all students and sessions: `list`
 
 Displays list of all students and all sessions in TutorBuddy.
 
-![list](images/command_list.png)
-
 Format: `list` <br>
 
+Example:
+1. Type in the command `list` and press <kbd>Enter</kbd>.
```
Comment (samleewy) [inline docs/UserGuide.md:253]: @enhao25's enter is lowercase. Maybe y'all can standardize? I personally think this Enter is better.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -250,14 +259,24 @@ Run the <code>clear</code> command to start working with TutorBuddy in a clean s
 
 Format: `clear`
 
+Example:
+1. Type in the command `clear` and press <kbd>Enter</kbd>.
+2. All data in TutorBuddy will be wiped clean.
+![clear_command](images/clear_command.png)
```
Comment (samleewy) [inline docs/UserGuide.md]: For this, i'm not sure if it shows much hahaha. Would it be better to show a before and after?

**Review by samleewy — COMMENTED** [review]
some comments! thanks for all the screenshotty stuff

File: docs/UserGuide.md (hunk 2 above)
Comment (enhao25) [reply docs/UserGuide.md:253]: Can! I will update from enter to Enter also!

File: docs/UserGuide.md (hunk 3 above)
Comment (JonahhGohh) [reply docs/UserGuide.md]: kinda agree with you. maybe i should just remove it?

Comment (samleewy) [reply docs/UserGuide.md]: actually yes, let's just remove it? haha

**Review by nowknowing — APPROVED** [review]
LGTM.
taking out clear seems fine too

**Pull request closed without merging**